mmtrack/models/reid/linear_reid_head.py

```diff
@@ -32,17 +38,38 @@ def __init__(self,
                  norm_cfg=None,
                  act_cfg=None,
                  num_classes=None,
-                 loss=dict(type='CrossEntropyLoss', loss_weight=1.0),
+                 loss_cls=dict(type='CrossEntropyLoss', loss_weight=1.0),
+                 loss_triplet=dict(
+                     type='TripletLoss', margin=0.3, loss_weight=1.0),
+                 cal_acc=False,
                  topk=(1, )):
-        super(LinearReIDHead, self).__init__(loss=loss, topk=topk)
+        super(LinearReIDHead, self).__init__(
+            loss=dict(type='CrossEntropyLoss'), topk=topk)
         self.num_fcs = num_fcs
         self.in_channels = in_channels
         self.fc_channels = fc_channels
         self.out_channels = out_channels
         self.norm_cfg = norm_cfg
         self.act_cfg = act_cfg
         self.num_classes = num_classes
-        self.loss = build_loss(loss)
+        self.compute_accuracy = Accuracy(topk=self.topk)
+        if not loss_cls:
+            if isinstance(num_classes, int):
```

Comment:
We do not need an argument to set whether calculate the acc, just cal it if we have ce loss